docs/api.md

```diff
@@ -354,6 +374,34 @@ const FocusableNode = () => (
 
 In the example above, the third node will get the default focus when mounting our page.
 
+# SpatialNavigationDeviceTypeProvider
```

Comment:
mention that you should only use the provider, so the internals are described here for science but won't be used

however, it is essential to inject it for the web pointers to work 😊

Reply:
Done :)